fetchcode/package.py

```diff
@@ -327,3 +327,30 @@ def get_rubygems_data_from_purl(purl):
         download_url=download_url,
         **purl.to_dict(),
     )
+
+
+@router.route("pkg:brew/.*")
+def get_homebrew_data_from_purl(purl):
+    """
+    Generate `Package` object from the `purl` string of rubygems type
+    """
+    purl = PackageURL.from_string(purl)
+    name = purl.name
+    base_path = "https://formulae.brew.sh/api/formula"
+    api_url = f"{base_path}/{name}.json"
+    response = get_response(api_url)
+    declared_license = response.get("license")
+    homepage_url = response.get("homepage")
+    version = response.get("versions")
+    download_path = response.get("urls")["stable"]["url"]
```

Comment:
`urls = response.get('urls') or {{}}
stable_url = urls.get('stable') or {}
download_url = stable_url.get('url')`

Reply:
Remove line 345-349 and use this

Reply:
oh ok, will replace that, sorry for these mistakes, will remember and learn continuing further. Thank you for the feedback